Shared/sdk/SharedUtil.Crypto.h

```diff
@@ -28,4 +31,48 @@ namespace SharedUtil
 
         return result;
     }
+
+    inline std::pair<SString, SString> Aes128encode(const SString& sData, const SString& sKey)
+    {
+        using namespace CryptoPP;
+
+        AutoSeededRandomPool rnd;
+
+        SString result;
+        SString sIv;
+
+        SecByteBlock key(0x00, AES::DEFAULT_KEYLENGTH);
+        SecByteBlock iv(0x00, AES::BLOCKSIZE);
```

Comment:
Why do you even use these? Just pass in the data directly from the `string`s.

Reply:
Fixed